src/test/kotlin/org/rust/lang/core/resolve/ref/RsLiteralWebReferenceProviderTest.kt

```diff
@@ -48,6 +48,16 @@ class RsLiteralWebReferenceProviderTest : RsTestBase() {
         }
     """, "http://localhost:8080", "https://github.com/foo/bar")
 
+    fun `test for invalid rust issue row`() = checkUrlReferences("""
+        #![unstable(issue = "0")]
+
+        #[unstable(feature = "foo", issue = "0")]
+        fn foo() { }
+
+        #[rustc_const_unstable(feature = "bar", issue = "0")]
+        fn bar() { }
```

Comment:
BTW let's better put carets at issue numbers in this case. Otherwise, the test will try to find references at wrong offset